lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -416,7 +416,7 @@ sub serial_console_diag_banner {
     # max_length - length of the text - 4x2 dividing spaces
     my $symbol_fill = ($max_length - length($input_text) - 8) / 2;
     $input_text = '#' x $symbol_fill . ' ' x 4 . $input_text . ' ' x 4 . '#' x $symbol_fill;
-    script_run($input_text, quiet => 1, die_on_timeout => 0, timeout => 1);
+    script_run($input_text, quiet => 1, timeout => 1);
```

Comment:
I will handle this part in separate PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19906
This was a very bad line of code so I will handle it properly with `enter_cmd` , plus there are unit tests that need adjustment. So you can omit this change completely. 
Thanks though.

Reply:
PR was merged, so this is fixed now.